Sprintfinity3902/Entities/Link/Player.cs

```diff
@@ -81,9 +81,16 @@ public void UseItem()
 
         public Rectangle getRectangle()
         {
-            return new Rectangle((int)X, (int)Y, 16*5, 16*5);
+            return new Rectangle((int)X, (int)Y, 16*Global.Var.SCALE, 16 * Global.Var.SCALE);
```

Comment:
This method is outdated.... And entirely my fault, I'll grab it with my next pr.